### ISSUE TITLE
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table, in Dashboard screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#887]
**Describe the bug**
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**To Reproduce**
1. Go to Dashboard screen (http://localhost:9992/cm/app/?view=dash)
2. Click on "Add Dashboard Gadgets" and select "Blogs" dashboard gadget.
3. If you right click on "Blog Title" and inspect on it, then its a table.
4. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**Expected behavior**
scope="col" for th tag and scope = "row" for very first td in each tr tag, in table, should be present.

**Additional context**
Navigate all the options available for similar issue and fix.
